app/models/SearchHits.ts

```diff
@@ -14,53 +18,107 @@ export interface ServiceHit
   extends Omit<Service, "schedule" | "recurringSchedule" | "instructions">,
     BaseHit {
   type: "service";
-  instructions: string[];
-  phones: PhoneNumber[];
+  instructions: string[] | [];
+  phones: PhoneNumber[] | [];
   recurringSchedule: RecurringSchedule | null;
-  resource_schedule: ScheduleDay[];
-  schedule: ScheduleDay[];
+  resource_schedule: ScheduleDay[] | [];
+  schedule: ScheduleDay[] | [];
   service_id: number;
   service_of: string;
 }
-
 export interface OrganizationHit
   extends Omit<Organization, "schedule" | "recurringSchedule">,
     BaseHit {
   type: "resource";
   schedule: ScheduleDay[];
   recurringSchedule: RecurringSchedule | null;
 }
-
+export type SearchResultsResponse = AlogliaSearchResultsType<SearchHit>;
 export type SearchHit = ServiceHit | OrganizationHit;
+export type TransformedSearchHit = Hit<
+  SearchHit & {
+    recurringSchedule: RecurringSchedule | null;
+    resultIndexDisplay: string;
+    markerTag: string;
+    longDescription: string;
+    path: string;
+    headline: string;
+    resource_path: string;
+    geoLocPath: string;
+    phoneNumber: string | null;
+    websiteUrl: string | null;
+  }
+>;
+export interface SearchMapHitData
+  extends AlogliaSearchResultsType<TransformedSearchHit> {
+  hits: TransformedSearchHit[];
+}
 
-/**
- * Transform Algolia search hits such that each hit has a recurringSchedule that
- * uses the time helper classes.
- */
-export const addRecurringScheduleToSeachHits = (hits: SearchHit[]) => {
-  const hitsWithRecurringSchedule = hits.flatMap((hit) => {
-    switch (hit.type) {
-      case "resource":
-        return {
-          ...hit,
-          recurringSchedule: hit.schedule?.length
-            ? parseAlgoliaSchedule(hit.schedule)
-            : null,
-        };
-      case "service": {
-        const schedule = hit.schedule || hit.resource_schedule;
-        return {
-          ...hit,
-          recurringSchedule: schedule?.length
-            ? parseAlgoliaSchedule(schedule)
-            : null,
-        };
-      }
-      default:
-        // The item is neither a service or resource and should be removed.
-        // A 0 element array removes the item from the mapped array
-        return [];
-    }
-  });
-  return hitsWithRecurringSchedule;
+// TODO: Determine if we need this code
+export const getRecurringScheduleForSeachHit = (
+  hit: SearchHit
+): RecurringSchedule | null => {
+  let result = null;
+
+  if (hit.type === "resource") {
+    result = hit.schedule?.length ? parseAlgoliaSchedule(hit.schedule) : null;
+  }
+
+  if (hit.type === "service") {
+    const schedule = hit.schedule || hit.resource_schedule;
+
+    result = schedule?.length ? parseAlgoliaSchedule(schedule) : null;
+  }
+
+  return result;
 };
+
+//
```

Comment:
```suggestion
// Returns a view model of search result data for use in downstream components
// Developers are encouraged to manage computed properties here rather than within presentational components
```